libappimage/CMakeLists.txt

```diff
@@ -51,28 +45,28 @@ SET(LIBAPPIMAGE_SRC
 ###
 # Build
 ###
-add_library(appimage ${LIBAPPIMAGE_SRC})
-target_compile_definitions(appimage
+add_library(libappimage ${LIBAPPIMAGE_SRC})
+target_compile_definitions(libappimage
     PRIVATE _FILE_OFFSET_BITS=64
     PRIVATE HAVE_LIBARCHIVE3=0
     PRIVATE VERSION_NUMBER="${VERSION_NUMBER}"
 )
-set_target_properties(appimage PROPERTIES VERSION ${VERSION_NUMBER})
-set_target_properties(appimage PROPERTIES PUBLIC_HEADER ./libappimage.h)
+set_target_properties(libappimage PROPERTIES PREFIX "")
+set_target_properties(libappimage PROPERTIES VERSION ${VERSION_NUMBER})
+set_target_properties(libappimage PROPERTIES PUBLIC_HEADER ./libappimage.h)
 
-target_link_libraries(appimage
-    ssl
-    squashfuse
-    crypto
-    z
-    lzma
+target_link_libraries(libappimage
     pthread
-    archive
-    inotifytools
+    ${squashfuse_LIBRARIES}
+    ${ZLIB_LIBRARIES}
+    ${libarchive_LIBRARIES}
+    ${OPENSSL_LIBRARIES}
```

Comment:
I'd love to get rid of this dependency yesterday - any chance?

Reply:
If we find out where it is used, we can probably get rid of it, yes.

Reply:
It is used for calculating the sha checksum that is needed as part of the gpg2 signing.

Reply:
We can probably just add an SHA-x algorithm in a header file somewhere, replacing that dependency.